modules/custom/az_core/az_core.module

```diff
@@ -52,6 +52,48 @@ function az_core_toolbar() {
     '#weight' => 101,
   ];
 
+  $siteurl = \Drupal::request()->getHost();
+
+  $items['az_quickstart']['tray'] = [
+    '#heading' => t('Quickstart links'),
+    'az_links' => [
+      '#theme' => 'links__toolbar_az_core',
+      '#links' => [
+        'docs' => [
+          'title' => t('Documentation'),
+          'url' => Url::fromUri('https://quickstart.arizona.edu'),
+          'attributes' => [
+            'title' => t('Arizona Quickstart 2 Documentation'),
+            'target' => t('_blank'),
+          ],
+        ],
+        'settings' => [
+          'title' => t('Settings'),
+          'url' => Url::fromRoute('az_core.az_quickstart'),
```

Comment:
This route (`az_core.az_quickstart`) was what the original `Quicsktart 2` toolbar item linked to (before it linked to the docs) but I think it might be better to link directly to the settings form itself.

(`/admin/config/az-quickstart/settings` instead of `/admin/config/az-quickstart`)
```suggestion
          'url' => Url::fromRoute('az_core.az_settings'),
```